src/rprblender/export/material.py

```diff
@@ -36,7 +36,7 @@ def get_material_output_node(material):
         return None
 
     return next((node for node in material.node_tree.nodes
-                 if node.bl_idname == 'ShaderNodeOutputMaterial' and node.is_active_output),
+                 if node.bl_idname == 'ShaderNodeOutputMaterial' and node.target == 'ALL'),
```

Comment:
Thinking about this, we should also accept Cycles target?

Reply:
In an instance where both exist, prioritizing all should always be first, unless there's a way to add prorender to the list of rendertargets without redefining the output class (so it doesn't ONLY show prorender in an output node placed while the renderer is set as prorender).

In any other case though really, it's dangerous to go with a renderer specific to any other renderer. If someone specifically specifies cycles, why should prorender use it?

Reply:
There is not a way to add ProRender to the list of outputs.

Generally speaking we try to translate eevee / cycles nodegraphs into something RPR can use.  The case that we're really talking about here if:
1.  The user does not have an "All" output node.
2.  The user has two output nodes, one for Cycles and one for EEVEE.  

In that case which should we choose to use?  There's no obvious correct answer.  So I'm ok with just the solution you have here.  

Of course you could do something like:
 Look for the first 'ALL" output node, if none found choose the first "Cycles" output, if not then whichever EEVEE.  
 
 Willing to hear opinions on this though.

Reply:
aha! I get what you mean now.

Perhaps when a non preferred output is taken it can warn a user in the log?

Just thinking about the case where I (and a handful of other people I know) run two sets of nodes per material for previews and renders, it would be nice to know if the preferred output was not available.